README.md

```diff
@@ -30,12 +30,12 @@ This team is spun off from the [Modules team](https://github.com/nodejs/modules)
 
 1. Refactor Node’s internal ESM loader to move its exception on unknown file types from within `resolve` (on detection of unknown extensions) to within `load` (if the resolved extension has no defined translator).
 
+1. Implement chaining as described in the [design](doc/design/proposal-chaining-middleware.md), where the `default<hookName>` becomes `next` and references the next registered hook in the chain.
+
 ### In Progress
 
 ### Upcoming
 
-1. Implement chaining as described in the [design](doc/design/proposal-chaining-middleware.md), where the `default<hookName>` becomes `next` and references the next registered hook in the chain.
-
 1. Get a `load` return value of `format: 'commonjs'` to work, or at least error informatively. See https://github.com/nodejs/node/issues/34753#issuecomment-735921348.
```

Comment:
Isn't this already done too?

Reply:
I think no because ESMLoader can't talk to CJSLoader or something. We could add an error (or maybe a warning?) though.

Reply:
If you or someone has time, we should confirm this one way or the other. I think I thought it was broken but then the test CoffeeScript loader got fixed, which proves that this wasn't an issue (but it was an author error on my part, which suggests maybe we need a better error message).